src/StackAdmin/Azs.Commerce.Admin/Module/Azs.Commerce.Admin/0.1.0/GeneratedHelpers.psm1

```diff
@@ -0,0 +1,119 @@
+<#
+The MIT License (MIT)
+
+Copyright (c) 2017 Microsoft
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+#>
+
+#########################################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+#
+# Licensed under the MIT license.
+#
+#########################################################################################
+Microsoft.PowerShell.Core\Set-StrictMode -Version Latest
+
+
+<#
+.DESCRIPTION
+   Creates delegating handlers for Microsoft Azure generated modules.
+#>
+function Get-AzDelegatingHandler
+{
+    [CmdletBinding()]
+    param()
+
+    New-Object -TypeName System.Net.Http.DelegatingHandler[] 0
+}
+
+<#
+.DESCRIPTION
+   Gets the Azure subscription ID from the logged in AzureRM context.
+#>
+function Get-AzSubscriptionId
+{
+    [CmdletBinding()]
+    param()
+
+    $AzureContext = & "Get-AzureRmContext" -ErrorAction Stop
+    if(Get-Member -InputObject $AzureContext.Subscription -Name SubscriptionId)
+    {
+        return $AzureContext.Subscription.SubscriptionId
+    }
+    else
+    {
+        return $AzureContext.Subscription.Id        
+    }
+}
+
+<#
+.DESCRIPTION
+   Gets the resource manager URL from the logged in AzureRM context.
+#>
+function Get-AzResourceManagerUrl
+{
+    [CmdletBinding()]
+    param()
+
+    $AzureContext = & "Get-AzureRmContext" -ErrorAction Stop    
```

Comment:
This is not a good way to get the resource manager url.  It shoudl aslo be unnecessary if the client is correctly constructed.

Reply:
It is auto-generated, I don't actually use it I believe.